releases.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Releases
+summary: Release notes for older versions of CockroachDB.
+toc: false
+---
+
+<style>
```

Comment:
To separate the content from the styling, how about adding this as `/css/components/_release.scss` and then adding `@import "components/release";` to `/css/customstyles.scss`?